lib/facilities/bulk_configuration.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'common/client/configuration/rest'
+require 'common/client/middleware/response/json_parser'
+require 'common/client/middleware/response/raise_error'
+
+module Facilities
+  class BulkJSONConfiguration < Common::Client::Configuration::REST
+    def connection
+      Faraday.new(base_path, headers: base_request_headers, request: request_options) do |conn|
+        conn.use :breakers
+        conn.request :json
+
+        conn.response :raise_error, error_prefix: service_name
+        conn.response :json_parser
+
+        conn.adapter Faraday.default_adapter
+      end
+    end
+  end
+
+  class AccessWaitTimeConfiguration < BulkJSONConfiguration
+    def base_path
+      Settings.locators.vha_access_waittime
+    end
+
+    def service_name
+      'VHA_Access_PWT'
+    end
```

Comment:
Total Nitpick, but these service names would be used by BackendServiceException to generate custom error messages from exception.en.yml -- With this name, you would end up with something like "VHA_ACCEESS_PWT135" as the error code. Which isn't a problem, just wanted you to be aware of it.